tests/ngui-e2e/server/README.md

```diff
@@ -1,159 +1,135 @@
-# NGUI E2E Server
+# Lightrail API with Next Gen UI Agent
```

Comment:
I don't like this name. First, Lightrail is RedHat internal platforms, we should not expose it too much. Second, this app can be used anywhere, only LlamaStack is required. And main base for the name should be use of this app, it is intended for E2E testing and preview purposes, so name should reflect this mainly.